content/en/docs/reference/scheduling/config.md

```diff
@@ -285,10 +260,17 @@ only has one pending pods queue.
 * A plugin enabled in a v1beta2 configuration file takes precedence over the default configuration for that plugin.
 
 * Invalid `host` or `port` configured for scheduler healthz and metrics bind address will cause validation failure.
+{{% /tab %}}
+
+{{% tab name="v1beta2 → v1beta3" %}}
+* Three plugins' weight are increased by default:
+  * `InterPodAffinity` from 1 to 2
+  * `NodeAffinity` from 1 to 2
+  * `TaintToleration` from 1 to 3
 
 {{% /tab %}}
-{{< /tabs >}}
 
+{{< /tabs >}}
```

Comment:
Nit: let's add a space after this line to give some space between it and the heading.


```suggestion
{{< /tabs >}}

```

Reply:
sounds good. done:)